cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -307,6 +308,7 @@ handleSimpleNode p trace nodeTracers npm onKernel = do
              vTr = appendName "version" tr
              cTr = appendName "commit"  tr
          traceNamedObject rTr (meta, LogMessage (show (ncProtocol nc)))
+         traceNamedObject rTr (meta, LogMessage ("NetworkMagic " <> show (unNetworkMagic $ nodeNetworkMagic (Proxy @blk) cfg)))
```

Comment:
These should always be the same. The distinction is just where they are used. We should get on with eliminating them as separate values. In Byron the both come from the genesis. In the Shelley genesis they're separate values, but that's a misunderstanding and a mistake.

Reply:
Output is:
```
TPraos
NetworkMagic X
```
so they are not the same.